Purchases/Networking/ETagManager.swift

```diff
@@ -57,8 +57,7 @@ class ETagManager {
                 return storedResponse
             }
             if retried {
-                Logger.warn(String(format: Strings.network.could_not_find_cached_response_in_already_retried,
-                        resultFromBackend.description))
+                Logger.warn(Strings.network.could_not_find_cached_response_in_already_retried(response: resultFromBackend.description))
```

Comment:
just noticed that this line goes over 120 characters and causes a warning from swiftlint, could you add a line break?

Reply:
Oops 🤦🏼‍♂️